<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>551</width>
      <height>231</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>547</width>
       <height>227</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_A">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_B">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_C">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_D">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_F">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_E">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_H">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_G">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_I">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_O">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>O</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_P">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>P</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_J">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>J</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_L">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>L</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_K">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_N">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>N</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_M">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_R">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Q">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Q</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_S">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_V">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_U">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>U</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_W">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>W</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Y">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_X">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Z">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Symbol_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Symbol_1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>,</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Symbol_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_T">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>T</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Symbol_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>!</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>162</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Enter">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>212</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Up">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>27</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Down">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>64</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>507</width>
       <height>77</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_1">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_2">
      <property name="geometry">
       <rect>
        <x>56</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_3">
      <property name="geometry">
       <rect>
        <x>106</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_5">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_6">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_4">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_10">
      <property name="geometry">
       <rect>
        <x>456</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_7">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_8">
      <property name="geometry">
       <rect>
        <x>356</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Dis_9">
      <property name="geometry">
       <rect>
        <x>406</x>
        <y>43</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>256</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Del">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Del</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Send">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
